Shared/Samples/Show geodesic sector and ellipse/ShowGeodesicSectorAndEllipseView.swift

```diff
@@ -0,0 +1,338 @@
+        private let sectorMarkerSymbol = SimpleMarkerSymbol(style: .circle, color: .green, size: 2)
+        private let ellipseLineSymbol = SimpleLineSymbol(style: .dash, color: .red, width: 2)
+        private let sectorFillSymbol = SimpleFillSymbol(style: .solid, color: .green)
+        
+        private var ellipseGraphic: Graphic!
+        private var sectorGraphic: Graphic!
+        
+        /// The direction (in degrees) of the ellipse's major axis.
+        @Published var axisDirection: Double = 45
+        /// Controls the complexity of the geometries and the approximation of the ellipse curve.
+        @Published var maxSegmentLength: Double = 1
+        /// Changes the sectors shape.
+        @Published var sectorAngle: Double = 90
+        /// Controls the complexity of the geometries and the approximation of the ellipse curve.
+        @Published var maxPointCount: Int = 1000
+        /// Changes the length of ellipse shape on one axis.
+        @Published var semiAxis1Length: Double = 200
+        /// Changes the length of ellipse shape on one axis.
+        @Published var semiAxis2Length: Double = 100
+        /// Changes the geometry type which the sector is rendered.
+        @Published var selectedGeometryType: GeometryType = .polygon
+        /// Changes the direction of the sector.
+        @Published var startDirection: Double = 45
+        
+        func refreshSector() {
+            guard let center = centerPoint else { return }
+            updateSector(center: center)
+        }
+        
+        private func updateSector(center: Point) {
+            ellipseGraphicOverlay.removeAllGraphics()
+            sectorGraphicOverlay.removeAllGraphics()
+            updateEllipse(center: center)
+            setupSector(center: center, geometryType: selectedGeometryType)
+        }
+        
+        private func setupSector(center: Point, geometryType: GeometryType) {
+            switch geometryType {
+            case .point:
+                // Generate sector as a multipoint (symbols)
+                var params = GeodesicSectorParameters<Multipoint>()
+                fillSectorParams(&params, center: center)
+                if let geometry = GeometryEngine.geodesicSector(parameters: params) {
+                    addSectorGraphic(geometry: geometry, symbol: sectorMarkerSymbol)
+                }
+            case .polyline:
+                // Generate sector as a polyline (outlined arc)
+                var params = GeodesicSectorParameters<Polyline>()
+                fillSectorParams(&params, center: center)
+                if let geometry = GeometryEngine.geodesicSector(parameters: params) {
+                    addSectorGraphic(geometry: geometry, symbol: sectorLineSymbol)
+                }
+            case .polygon:
+                // Generate sector as a filled polygon
+                var params = GeodesicSectorParameters<Polygon>()
```

Comment:
Make sure to test your code on all supported platforms (iOS, Mac Catalyst, and visionOS). When you do, you'll find that this line has an error when building for Mac Catalyst as macOS has a type named `Polygon`. The solution is to prefix the type with the enclosing module: `ArcGIS.Polygon`.

Reply:
Updated to GeodesicSectorParameters<ArcGIS.Polygon>